raiden-ts/src/raiden.ts

```diff
@@ -1011,6 +1021,53 @@ export class Raiden {
         .reduce((acc, iou) => acc.add(iou.amount), Zero);
     return balance.sub(owedAmount);
   }
+
+  /**
+   * Deposits the amount to the UserDeposit contract with the target/signer as a beneficiary.
+   * The deposited amount can be used as a collateral in order to sign valid IOUs that will
+   * be accepted by the Services.
+   *
+   * Throws an error, in the following cases:
+   * <ol>
+   *  <li>The amount specified equals to zero</li>
+   *  <li>The target has an insufficient token balance</li>
+   *  <li>The "approve" transaction fails with an error</li>
+   *  <li>The "deposit" transaction fails with an error</li>
+   * </ol>
+   *
+   * @param amount - The amount to deposit on behalf of the target/beneficiary.
+   * @returns transaction hash
+   */
+  public async depositToUDC(amount: BigNumberish): Promise<Hash> {
+    const depositAmount = bigNumberify(amount);
+
+    if (depositAmount.isZero()) throw new Error('Please deposit a positive amount.');
+
+    const { userDepositContract, address } = this.deps;
+
+    const tokenAddress = await this.userDepositTokenAddress();
+    const serviceToken = CustomTokenFactory.connect(tokenAddress, this.deps.signer);
```

Comment:
`CustomToken` is needed only for minting. As we don't mint on this function, please, use `HumanStandardToken` for consistency